src/menus/icon-menu.jsx

```diff
@@ -114,12 +108,6 @@ const IconMenu = React.createClass({
      * horizontal: [left, center, right].
      */
     targetOrigin: PropTypes.origin,
-
-    /**
-     * Sets the delay in milliseconds before closing the
-     * menu when an item is clicked.
-     */
-    touchTapCloseDelay: React.PropTypes.number,
```

Comment:
Can someone explain what the purpose of this prop was for? I see that `<AutoComplete /> still has a prop for this but I don't know why there a delay was configurable in the first place.